src/main/java/net/minecraftforge/fmp/microblock/MicroblockDelegate.java

```diff
@@ -0,0 +1,174 @@
+package net.minecraftforge.fmp.microblock;
+
+import com.google.common.base.Optional;
+
+import net.minecraft.block.Block;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraftforge.fmp.multipart.IMultipart;
+import net.minecraftforge.fmp.multipart.Multipart;
+
+/**
+ * Represents an object to which microblock events are delegated. This allows for custom interaction and data storage
+ * inside the microblock.
+ */
+public class MicroblockDelegate
```

Comment:
Why is this class needed? All it does is null everything except sendUpdatePacket?